pkg/cmd/component/bootstrap_new.go

```diff
@@ -23,10 +24,13 @@ import (
 //testing changes
 const (
 	BootstrapRecommendedCommandName = "bootstrap-new"
-	componentNameFlag               = "component-name"
-	appliactionNameFlag             = "application-name"
-	gitRepoURLFlag                  = "git-repo-url"
-	secretFlag                      = "secret"
+
+	componentNameFlag     = "component-name"
+	applicationNameFlag   = "application-name"
+	gitRepoURLFlag        = "git-repo-url"
+	secretFlag            = "secret"
```

Comment:
Should this be the tokenFlag?

Reply:
Did you mention that you will change it in the bootstrap command, right?

Reply:
Done